app/views/papers/_tags_form.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="tags-form hidden">
+  <%= form_for(paper, html: {class: 'form-inline'}) do |f| %>
+    <div class="form-group col-md-12">
+      <%= f.text_field :tag_list, value: paper.tag_list.join(","), placeholder: "No tags added yet...", class: "form-control input-sm" %>
+      <%= f.submit 'Add tags', class: "btn btn-xs btn-primary" %>
+      <%= link_to "cancel", '#', class: "cancel-add-tag link-btn" %>
+    </div>
+  <% end %>
+  <i class='help-block'><small>Tags should be seperated by commas.</small></i>
+</div>
```

Comment:
you mean like this?

![image](https://cloud.githubusercontent.com/assets/2272356/20247935/037d42c0-a99f-11e6-9c06-8bbe77802bb6.png)

This is definitely where I'm hoping this will end up.  I think that would involve some more ajaxing so I will pass on this for now.  I really like the definitions and autocomplete feature though.  I'm also still keen on the idea of colors for methodology and biases too.  I think the colors would stay consistent across the site (blue for methodology, red for biases) but people could choose which it is for the tag.  Just a thought.